packages/adapter/src/request-building/build-request.test.ts

```diff
@@ -1,5 +1,5 @@
 import * as build from './build-request';
-import * as fixtures from 'test/fixtures';
+import * as fixtures from '../../test/fixtures';
```

Comment:
I was previously mixing these kinds of imports. I think it's better to have them all consistent